contracts/staking/STWBulkHarvest.sol

```diff
@@ -0,0 +1,13 @@
+pragma solidity ^0.8.4;
+
+import "./StakingTokenWrapper.sol";
+
+/// @notice contract to bulk harvest multiple Staking Token Wrappers in a single transaction
+/// stateless contract with no storage and can only call harvest on the STW's
+contract STWBulkHarvest {
+    function bulkHarvest(StakingTokenWrapper[] calldata stw) external {
```

Comment:
It would be cool to expose a method for harvesting a canonical list from storage too I think

Reply:
that costs way more gas because you have to do a SLOAD for each address you pull out of storage.

Reply:
If it pulls from storage, then you would need access controls, and have to permission functions to append or delete from the list. This contract should be stateless to keep it simple.